Build/libHttpClient.Android/app/build.gradle

```diff
@@ -57,7 +57,7 @@ android {
 
     libraryVariants.all { variant ->
         variant.outputs.each { output ->
-            output.outputFileName = "../../../../../../Binaries/Android/java/libHttpClient-${variant.name}.aar"
+            output.outputFileName = "../../../../../../Binaries/Android/java/${variant.getFlavorName()}/libHttpClient-${variant.getBuildType().getName()}.aar"
```

Comment:
could we get the aars to be put next to the .so?

Reply:
There is no `.so`, you mean the `.a`?

Currently the C++ binaries go into:
```
Binaries/Android/native/<debug|release>/<abi>/libNAME.a
```
and the Java binary goes to:
```
Binaries/Android/java/libHttpClient-<abi><debug|release>.aar
```

Since for the `.aar` the <abi> is meaningless (the fact that it's there is an artifact of how the Gradle files are structured), this change would put it in:
```
Binaries/Android/java/<abi>/libHttpClient-<debug|release>.aar
```

So while they're not directly alongside the `.a`, everything is included under `Binaries/Android`.

Reply:
I can align the Java and native lib paths to have the <abi> and <debug|release> in the same order, and/or take the <debug|release> out of the filename for the Java or something (or put it into the filename for the native libs). Open to feedback here.

Reply:
per offline conversation, i'll align to have both native and java binaries in:
```
Binaries/Android/<abi>/<debug|release>
```